Azure-Samples/chat-with-your-data-solution-accelerator
# refactor: Make backend orchestrator, parser, and  plugins PEP8

## Purpose



* Make backend orchestrator, parsers and plugins PEP8
* Resolves #856 
* Resolves #857 

## Does this introduce a breaking change?



## How to Test
* Run the tests:
```
poetry run pytest -m "not azure and not functional"    
poetry run pytest -m "functional"  
```
* Run the application locally and go to http://localhost:8502/Ingest_Data
* Upload a document on the above endpoint
* Verify that the content is available.

## What to Check
* Verify that the content is available.

![SCR-20240513-kvtl-2](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/4739534/f02fa111-3893-4886-85c6-ee3f0843bb77)
